src/main/java/com/jmb/springfactory/model/entity/User.java

```diff
@@ -28,6 +28,8 @@
 
   @NotNull
   private String name;
+  
+  private String password;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "password" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)